packages/ember-metal/lib/tags.js

```diff
@@ -19,11 +21,49 @@ export function tagFor(object, _meta) {
   }
 }
 
+class SchedulerRegistrar {
+  constructor() {
+    let schedulerRegistrar = this;
+    this._eventCallbacks = {
+      begin: []
+    };
+
+    this._trigger = backburner._trigger;
+    this.register = backburner.on;
+    this.deregister = backburner.off;
+
+    function bindEventName(eventName) {
+      return function trigger(arg1, arg2) {
+        schedulerRegistrar._trigger(eventName, arg1, arg2);
+      };
+    }
+
+    for (let eventName in this._eventCallbacks) {
+      if (backburner._eventCallbacks.hasOwnProperty(eventName)) {
+        backburner.on(eventName, bindEventName(eventName));
+      }
+    }
+  }
+
+  hasRegistrations() {
+    return !!this._eventCallbacks.length;
+  }
+}
+export const schedulerRegistrar = new SchedulerRegistrar();
```

Comment:
Defining this in `ember-metal/tags` bothers me. Is there a better way to do this?